com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -260,6 +262,9 @@ public static UniversalRenderPipelineAsset Create(ScriptableRendererData rendere
             // Initialize default Renderer
             instance.m_EditorResourcesAsset = instance.editorResources;
 
+            // Only enable for new URP assets by default
+            instance.m_ConservativeEnclosingSphere = true;
```

Comment:
@Verasl @phi-lira @eh-unity This is how I am enabling it only for new projects. Let me know if this is the right approach.